packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -15,7 +15,7 @@ import theme from '../../theme'
 import CommerceAPI from '../../commerce-api'
 import {
     BasketProvider,
-    CommerceAPIProvider,
+    CommerceAPIProvider as _CommerceAPIProvider,
```

Comment:
This is just a process question about how we do things on the team. Since we are protected behind release versions and assume people are consuming our code via NPM, do we regularly commit "WIP" code like this, which is transitional in nature? Is there some special signal that lets us know some incomplete work is still in `develop` and we should hold the next release? Or is this just in here because the PR is "draft" and therefore we know this will get cleaned up before the PR truly merges?

Reply:
@bfeister I can't say about how things are done in general.. but for this work on integrating the commerce hooks, we plan to use a long-lived branch "feature/integrate-commerce-sdk-react". So this kind of transitional code will be cleaned up by the time it reaches the develop branch.

Reply:
Gotcha! Yeah, pointed at the feature branch for now, I missed that. 👍